tests/Unit/Evolution/DiscontinuousGalerkin/Test_MortarData.cpp

```diff
@@ -137,7 +139,9 @@ void test_global_time_stepping_usage() {
 
   check_serialization(make_not_null(&mortar_data));
 
-  mortar_data = MortarData<Dim>{};
+  CHECK(mortar_data.current_buffer_index() == 0);
+  mortar_data.next_buffer();
+  CHECK(mortar_data.current_buffer_index() == 1);
```

Comment:
I can add some checks for this in this function. Since MortarData isn't responsible for cleaning up anything, I think the test can be simple and just check that the values are what we expect.